modules/host/consts.go

```diff
@@ -76,6 +76,10 @@ var (
 	// opportunity cost in being a host.
 	defaultCollateral = types.NewCurrency64(50) // 50 SC / GB / Month
 
+	defaultCollateralFraction = types.NewCurrency64(650e3)
+
+	defaultMaxCollateral = types.NewCurrency64(10000).Mul(types.SiacoinPrecision)
```

Comment:
Given that requesting collateral from the host is so expensive, especially when file contracts are only 12 weeks long (the renter pays a 3.9% fee to lock the host out of funds for 12 weeks? that's a really expensive attack, especially since the renter has to lock itself out of half the funds for 12 weeks as well), I don't think we really need to worry about attackers here. It's just too expensive to be practical, and the host can get around the problem simply by getting more coins. The host is not losing money at all, it'll come back, which means that the only risks here are potential profit from interest, and the risk inherent to holding a cryptocurrency.

A bigger fear would be that we have a lot of renters who try out early on, lock up a bunch of host funds for a few weeks, but then never use the storage because they get disinterested and go back to DropBox. So, maybe early on it makes sense to have a small default max collateral, but I think that as time passes I think we'll be pretty easily able to jump it quite a bit.